device/arista/x86_64-arista_7800r3a_36d2_lc/Arista-7800R3A-36D2-C36/fabric_monitor_config.json

```diff
@@ -4,7 +4,7 @@
       "monErrThreshRxCells": 61035156,
       "monPollThreshRecovery": 8,
       "monPollThreshIsolation": 1,
-      "monCapacityThreshWarn": 100,
+      "monCapacityThreshWarn": 62,
```

Comment:
if only 62% of the links are up, will the chassis see any data plane impact? If yes, can we set the Warning threshold 10% higher so that the chassis can be safely removed from production before any impact is seen?

Reply:
> if only 62% of the links are up, will the chassis see any data plane impact? If yes, can we set the Warning threshold 10% higher so that the chassis can be safely removed from production before any impact is seen?

@vmittal-msft any thought ?  I am ok with setting any value here, just let me know.

Reply:
We shouldn't see any impact as 62% links should allow 75% line rate traffic in fully populated chassis. So i guess it is  fine.